app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -160,13 +160,21 @@ class MetadataManualInput extends React.Component {
   };
 
   autoPopulateMetadata = () => {
+    const { samples } = this.props;
     const { metadataFieldsToEdit, headersToEdit } = this.state;
 
+    let newMetadataFieldsToEdit = {};
+
+    // For each sample, merge auto-populate fields into existing fields (which may be empty).
+    // Existing fields take precedence.
+    samples.forEach(sample => {
+      newMetadataFieldsToEdit[sample.name] = merge(
+        AUTO_POPULATE_FIELDS,
+        metadataFieldsToEdit[sample.name] || {}
+      );
+    });
+
     const newHeadersToEdit = union(headersToEdit, keys(AUTO_POPULATE_FIELDS));
-    const newMetadataFieldsToEdit = mapValues(
```

Comment:
The previous code applied a change to values that were already in `metadataFieldsToEdit`. But unless something makes a change to metadata (such as setting default fields for the water control field), there will be no values in `metadataFieldsToEdit`, so the previous code would do nothing.

The new code iterates through the names of the samples that are being uploaded.